src/app/news/news-list/news-list.component.html

```diff
@@ -9,16 +9,20 @@ <h4>Всі новини</h4>
       <div class="col">
          <div class="card-columns">
             <div *ngFor="let one of news" class="card mb-0 mb-sm-2 shadow">
-               <ul class="list-group list-group-flush">
+               <ul class="list-group list-group-flush" style="height: 46px">
                   <li class="list-group-item">Категорія: {{ one.tournament ? one.tournament.title : 'Інше' }}</li>
                </ul>
-               <img class="card-img-top img-fluid p-sm-2" src="{{ newsImagesUrl }}/{{ one.image }}" alt="{{ one.title }}" />
-               <div class="card-body">
-                  <h4 class="card-title">{{ one.title }}</h4>
-                  <a routerLink="/news/{{ one.id }}" class="btn btn-outline-primary">Читати далі</a>
-                  <p class="card-text">
-                     <small class="text-muted">{{ one.created_at | time: 'fromNow' }}</small>
-                  </p>
+               <img class="card-img-top p-sm-2" src="{{ newsImagesUrl }}/{{ one.image }}" alt="{{ one.title }}" height="220" />
+               <div class="p-2 px-lg-3 py-lg-2 d-flex flex-column justify-content-between" style="height: 180px">
+                  <h4 class="card-title h4 d-block d-sm-none">{{ one.title }}</h4>
+                  <h4 class="card-title h6 d-none d-sm-block d-lg-none">{{ one.title }}</h4>
+                  <h4 class="card-title h5 h5 d-none d-lg-block">{{ one.title }}</h4>
+                  <div>
+                     <a routerLink="/news/{{ one.id }}" class="btn btn-outline-primary">Читати далі</a>
+                     <p class="card-text">
+                        <small class="text-muted">{{ one.created_at | time: 'fromNow' }}</small>
+                     </p>
+                  </div>
```

Comment:
It looks better than it was, but still has a lot of issues (some of them are introduced with this change):
Static width distorts image (image width 575px):
<img width="590" alt="Screenshot 2024-10-14 at 19 49 29" src="https://github.com/user-attachments/assets/8103b143-5e1e-413c-9889-da3d4128534b">

Images are cropped:
<img width="781" alt="Screenshot 2024-10-14 at 19 53 50" src="https://github.com/user-attachments/assets/e49f28d0-5c19-44aa-be1f-7e3ad1ee1ffc">

In my opinion adding static widths/heights don't work well with bootstrap responsive design.